packages/terra-aggregator/package.json

```diff
@@ -37,7 +37,7 @@
     "terra-arrange": "^2.19.0",
     "terra-button": "^2.0.1",
     "terra-content-container": "^2.4.0",
-    "terra-disclosure-manager": "^2.14.0",
+    "terra-disclosure-manager": "2.13.0",
     "terra-doc-template": "^1.1.0",
     "terra-form": "^2.0.1",
```

Comment:
how are we using `terra-form`? should probably remove it anyway since it's deprecated

Reply:
It's being used in the examples. I will update this in a subsequent PR; terra-aggregator will have more changes once this PR is merged.

Reply:
Be gentile! That's still my baby, even though it's getting removed!!!